modules/context/repo.go

```diff
@@ -897,7 +931,14 @@ func RepoRefByType(refType RepoRefType, ignoreNotExistErr ...bool) func(*Context
 		if len(ctx.Params("*")) == 0 {
 			refName = ctx.Repo.Repository.DefaultBranch
 			if !ctx.Repo.GitRepo.IsBranchExist(refName) {
-				brs, _, err := ctx.Repo.GitRepo.GetBranchNames(0, 0)
+				// get the first branch
+				brs, err := git_model.FindBranchNames(ctx, git_model.FindBranchOptions{
```

Comment:
Is it possbile to return empty because the data is not synchronized？

Reply:
done.